charts/civil-service/values.aat.template.yaml

```diff
@@ -61,7 +61,7 @@ java:
     FEIGN_CLIENT_CONFIG_REMOTEREPOSITORYSERVICE_URL: http://${SERVICE_NAME}-camunda/engine-rest/
     SERVICE_REQUEST_UPDATE: http://${SERVICE_FQDN}/service-request-update
     TESTING_SUPPORT_ENABLED: true
-    POLLING_EVENT_EMITTER_ENABLED: false
+    polling.event.emitter.enabled: false
```

Comment:
Earlier name is a better naming convention for env vars. Spring Relaxed binding should honour ur earlier value and infact the the recommended naming convention (Upper case with _ )

Reply:
agreed Praveen, then this will require further mapping of the property, will do